MediaBrowser.MediaEncoding/Probing/ProbeResultNormalizer.cs

```diff
@@ -280,8 +280,8 @@ private string NormalizeFormat(string format, IReadOnlyList<MediaStream> mediaSt
                     splitFormat[i] = "mpeg";
                 }
 
-                // Handle MPEG-2 container
-                else if (string.Equals(splitFormat[i], "mpeg", StringComparison.OrdinalIgnoreCase))
+                // Handle MPEG-TS container
+                else if (string.Equals(splitFormat[i], "mpegts", StringComparison.OrdinalIgnoreCase))
```

Comment:
The description and PR title mentions `mpeg-ts`. I just want to confirm that is `mpegts`?

Reply:
Yes. They are the same thing but the ffmpeg short name is mpegts.